src/sentry/demo/data_population.py

```diff
@@ -1252,37 +1256,39 @@ def populate_sessions(self, project, distribution_fn_num: int, mobile: bool, err
                     dsn, timestamp, mobile, version, num_versions, seen_versions, weights
                 )
 
-            # send sessions for duration info
-            session_data = {
-                "init": True,
-            }
-            self.send_session(sid, transaction_user["id"], dsn, timestamp, version, **session_data)
-            release_num = int(version.split(".")[-1])
-            threshold = rate_by_release_num[release_num]
             outcome = random.random()
-            if outcome > threshold:
-                if error_file:
-                    local_event = copy.deepcopy(error)
-                    local_event.update(
-                        project=project,
-                        platform=project.platform,
-                        timestamp=timestamp,
-                        user=transaction_user,
-                        release=version,
-                    )
-                    update_context(local_event)
-                    self.fix_error_event(local_event)
-                    self.safe_send_event(local_event)
-
-                data = {
-                    "status": "crashed",
-                }
-            else:
-                data = {
-                    "status": "exited",
-                }
+            if outcome > ind_session_threshold:
+                # send sessions for duration info
+                session_data = {"init": True}
+                self.send_session(
+                    sid, transaction_user["id"], dsn, timestamp, version, **session_data
+                )
+                release_num = int(version.split(".")[-1])
+                threshold = rate_by_release_num[release_num]
+                outcome = random.random()
+                if outcome > threshold:
```

Comment:
@roggenkemper i'm a bit confused why we are checking the `outcome` twice, seems odd

Reply:
the first time is to see if we are going to send a session or not, the second time is to determine if the session crashes or not.

Reply:
@roggenkemper let's just use a different distribution function then I think

Reply:
on second thought, that might be a bigger refactor. I guess what you have is OK if not ideal.